x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_details_content.tsx

```diff
@@ -95,7 +100,10 @@ export const EndpointDetailsContent = memo<EndpointDetailsContentProps>(
               />
             </ColumnTitle>
           ),
-          description: (
+          // TODO: 8.15 remove `EndpointAgentStatus` when `agentStatusClientEnabled` FF is enabled
```

Comment:
As above: 
I would suggest to remove usage of EndpointAgentStatusById when the agentStatusClientEnabled FF gets **removed**, not **enabled**.